src/routes/+page.svelte

```diff
@@ -1,2 +1,42 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<h1>Welcome to Kendo Club!</h1>
+
+<h2>What is Kendo?</h2>
+<p>
+	Kendo is a Japanese martial art which literally translates to “The Way of the Sword”. It's an
+	athletic sport which is played by means of one-on-one striking between opponents using shinai
+	(sword made from bamboo slats) while wearing kendo-gu (protective armor).
+</p>
+
+<h2>Who are we?</h2>
+
+<p>
+	Kendo Club at the University of Michigan is led by students and instructed by highly experienced
+	coaches. Our purpose is to provide an open and inviting opportunity for University of Michigan
+	students and local community members to learn and practice Kendo. We invite everyone of any skill
+	level to join us!
+</p>
+
+<h2>How do you join?</h2>
+
+<h3>Come to practice!</h3>
+<ul>
+	<li>
```

Comment:
I think there is some useful information on the join us page of the old website that I don't see here is that going to be somewhere else?

- no prior experience needed 
- $55 membership fee
- rec center membership
- new members required to purchase equipment

Reply:
Fixed in [c2b2351](https://github.com/kendoclub-at-umich/website/pull/31/commits/c2b2351ddf9ee7b99fe0039d1908f9a90a0fd38e)
The no prior experience needed is somewhat implied at the end of the "who are we" section, but I agree it should be clearly stated and prominent. 
The rest of those should absolutely  be included, thanks. I thought about it, and then I chose not to at the time for the sake of just having **something** that was done, and also bc I wanted to think about whether or not there would be a better place to put them.

Reply:
Yeah it's always nice to have something to iterate on instead of trying to get everything on the first pass